src/frontend/js/components/DashboardModerators/ListModerators/index.spec.tsx

```diff
@@ -0,0 +1,87 @@
+import 'regenerator-runtime/runtime';
+import { fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { User } from '../../../types/User';
+import { ContextFactory } from '../../../utils/test/factories';
+
+const myProps = {
+  users: [
+    { public_username: 'Thérèse', id: 1 },
+    { public_username: 'Thomas', id: 2 },
+    { public_username: 'Sam', id: 3 },
+  ],
+  totalUsers: 5,
+  setUser: (user: User) => {
+    console.warn(
+      `setUser has been called for user ${user.id} ${user.public_username}`,
+    );
+  },
+  onChange: () => {
+    console.warn('OnChange has been called for sure!');
+  },
+};
+
+describe('<ListModerators />', () => {
+  const contextProps = ContextFactory().generate();
+  (window as any).__ashley_frontend_context__ = { context: contextProps };
+  const ListModerators = require('.').default;
```

Comment:
Why are you using `require` here ? Why not adding `import ListModerators from '.';` with the other `import` ?